requirements.txt

```diff
@@ -25,5 +25,4 @@ pytest >= 3.3
 
 # Packaging tools
 PyInstaller ~= 3.3
-patchelf-wrapper~=1.0
```

Comment:
Can't use staticx without patchelf-wrapper. Could you remind me please why did it have to go?

Reply:
patchelf-wrapper is not compatible with Windows. I removed it from requirements-dev-windows but left it on requirements-dev-linux. It is used only in build_linux.sh. We'll need solve this if we want to make a build script on windows.